docs/examples/quick-start-with-terraform.md

```diff
@@ -0,0 +1,302 @@
+# Quick Start with Terraform
+
+This tutorial aims at presenting how to set up an environment with Liqo installed via Terraform and explore its most notable capabilities.
+You will create a _virtual cluster_ by peering two Kubernetes clusters and deploy a simple application on it.
+
+## Provision the infrastructure
+
+Before starting the tutorials below, you should ensure the following software is installed on your system:
+
+- [**Docker**](https://docker.io), the container runtime.
+- [**Kubectl**](https://kubernetes.io/docs/tasks/tools/install-kubectl/), the command-line tool for Kubernetes.
+- [**Terraform**](https://developer.hashicorp.com/terraform/tutorials/aws-get-started/install-cli), the command-line tool for Terraform.
+
+Then, let's open a terminal on your machine and launch the following script, which creates the complete infrastructure with all playground already set up.
+
+{{ env.config.html_context.generate_clone_example_tf('quick-start-terraform') }}
```

Comment:
Not clear to me what you mean with "complete infrastructure with all playground already set up". More details (of, even better, a picture) can be helpful here. Otherwise I know that the script creates something, but I have no idea what.